thirdparty/rr-full/plugins/eventlogs.pl

```diff
@@ -57,9 +57,11 @@ sub pluginmain {
 			::rptMsg($win_path);
 			::rptMsg("LastWrite Time ".gmtime($win->get_timestamp())." (UTC)");
 			my $cn;
-			if ($cn = $win->get_value("ComputerName")->get_data()) {
-				::rptMsg("ComputerName = ".$cn);				
-			}
+            if (defined($win->get_value("ComputerName"))) {
```

Comment:
Are these and the changes in all other .pl files still part of the upgrade or are these yours?

Reply:
These changes are fixes to undefined values that cause an error in the plugin and it will not completely execute.  These are separate from the upgrade PR.

Reply:
Okay, should these changes also be submitted to the RegRipper project? Reason I ask is that if we ever perform an upgrade these changes will get lost or need to be redone.

Reply:
Regripper 2.8 is no longer being supported as of 5/28/2020 according to this http://windowsir.blogspot.com/2020/05/regripper-v30.html so adding a PR to it will never happen as this is end of life for this version we are using.